src/_data/vbe.yml

```diff
@@ -226,9 +340,35 @@ extensions:
           2.3.4: not supported
           2.3.5-p1: not supported
           2.4.0: not supported
+          2.4.1: not supported
+          2.4.2: not supported
   -
     name: Vertex
     versions:
+      -
+        name: t.b.d
```

Comment:
Can you confirm that this should be released as is?

Reply:
They're placeholder values and will be updated prior to GA.